panther_analysis_tool/main.py

```diff
@@ -97,6 +99,7 @@
 POLICIES_PATH_PATTERN = "*policies*"
 QUERIES_PATH_PATTERN = "*queries*"
 RULES_PATH_PATTERN = "*rules*"
+TMP_HELPER_MODULE_LOCATION = os.path.join(tempfile.gettempdir(), "globals")
```

Comment:
`globals` may be a bit generic. Consider using a namespace (additional directory), like `panther-pat` or similar.